common-libs/common/test/factory/resolveDidResponse.ts

```diff
@@ -0,0 +1,98 @@
+export const did = 'did:jolo:569d3f3c3fb5f43568a6fd615f1e09dee169bf98b201813d7f998e241be7ff0e'
```

Comment:
please find other examples, jolo is being deprecated, so better to not add new mentions of that. polygon should be good